src/main/java/at/hannibal2/skyhanni/utils/StringUtils.kt

```diff
@@ -44,7 +44,7 @@ object StringUtils {
     /**
      * From https://stackoverflow.com/questions/10711494/get-values-in-treemap-whose-string-keys-start-with-a-pattern
      */
-    fun <T> subMapWithKeysThatAreSuffixes(prefix: String, map: NavigableMap<String?, T>): NavigableMap<String?, T>? {
+    fun <T> subMapWithKeysThatAreSuffixes(prefix: String, map: NavigableMap<String?, T>): Map<String?, T>? {
```

Comment:
Any reason for this change? NavigableMap is an interface that provides additional functionality and guarantees over Map, so this feels like a bit of a waste.